### ISSUE TITLE
The Spinner is keep Spinning after PopUp

### ISSUE THREAD

**Issue by dheerajbalmuri** [AhmadiRamin/office-addins-msal#1]
Hi Ahmad,

First of all Very Good job on combining pieces together. I'm having issue while setting it up . The Login Function isn throwing Popup and after i sign in the spinner keeps spinning instead of redirecting to  welcome screen. I just cloned your code and updated my ClineId, SharePoint URL. I tried to debug it but i believe it's failing in this method.

const spTokenResponse = await userAgentApp.acquireTokenSilent(spReqObj);

**Comment by AhmadiRamin** [comment]
Hi Dheeraj,

Did you follow the steps carefully? did you try to debug it to see if any errors come up?

Kind regards,
Ramin
________________________________
From: Dheeraj ***@***.***>
Sent: Thursday, June 3, 2021 8:40 PM
To: AhmadiRamin/office-addins-msal ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [AhmadiRamin/office-addins-msal] The Spinner is keep Spinning after PopUp (#1)


Hi Ahmad,

First of all Very Good job on combining pieces together. I'm having issue while setting it up . The Login Function isn throwing Popup and after i sign in the spinner keeps spinning instead of redirecting to welcome screen. I just cloned your code and updated my ClineId, SharePoint URL.

**Comment by dheerajbalmuri** [comment]
yes sir, i did follow the steps. It's rejecting the promise in this line of code
const spTokenResponse = await userAgentApp.acquireTokenSilent(spReqObj);
with this description ""Seamless single sign on failed for the user. This can happen if the user is unable to access on premises AD or intranet zone is not configured correctly Trace ID: d95bc968-3961-443c-acb7-42f0b3a74103".

**Comment by AhmadiRamin** [comment]
There are some solutions in the following links:

Get Login Error "Seamless single sign on failed for the user" for some users · Issue #1986 · AzureAD/microsoft-authentication-library-for-js (github.com)<https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/1986>

Receiving error - Seamless single sign on failed for the user · Issue #2372 · AzureAD/microsoft-authentication-library-for-js (github.com)<https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/2372>

Kind regards,
Ramin
________________________________
